src/components/Search.js

```diff
@@ -1,14 +1,159 @@
+    );
+    const tagsFilter = Object.keys(tags).filter((tag) => tags[tag] === true);
+
+    const url = `/projects?skill[]=${encodeURIComponent(
+      skillsFilter
+    )}&tag[]=${encodeURIComponent(tagsFilter)}&search=${encodeURIComponent(
+      searchQuery
+    )}`;
+    axiosInstance.get(url).then((res) => {
+      setProjects(res.data);
+    });
+  };
+
+  const navigateToProject = (projectId) => {
+    navigate(`/projects/${projectId}`);
+  };
+
+  useEffect(() => {
+    getProjects();
+  }, []);
+
+  useEffect(() => {
+    const { skills, tags } = queryParams;
+    (!isEmpty(skills) || !isEmpty(tags)) && handleSearch();
+  }, [queryParams]);
 
   return (
-    <>
-    
-      <Searchbar />
-    </>
+    <div className='search-page'>
+      <div className='skills-tag-container'>
+        {/* Skills card */}
+        {!!skills?.length && (
+          <div className='skill-tag-card'>
+            <h2>Skills</h2>
+            {skills?.map((skill, index) => (
+              <div key={`tag-${index}`} className='project-form-element'>
+                <input
+                  id='skills'
+                  name={skill}
+                  type='checkbox'
+                  onChange={handleInputChange}
+                />
+                <label htmlFor={skill}>{skill}</label>
+              </div>
+            ))}
+          </div>
+        )}
+
+        {/* Tags card */}
+        {!!tags?.length && (
+          <div className='skill-tag-card'>
+            <h2>Tags</h2>
+            {tags?.map((tag, index) => (
+              <div key={`tag-${index}`} className='project-form-element'>
+                <input
+                  id='tags'
+                  name={tag}
+                  type='checkbox'
+                  onChange={handleInputChange}
+                />
+                <label htmlFor={tag}>{tag}</label>
+              </div>
+            ))}
+          </div>
+        )}
+      </div>
+
+      {/* Search and projects */}
+      <div className='search-projects'>
+        <div className='searchbar'>
+          <input
+            type='text'
+            value={searchQuery}
+            className='form-control'
+            id='search'
+            placeholder='Search...'
+            onChange={({ target: { value } }) => setSearchQuery(value)}
+          />
+          <button className='btn btn-primary' onClick={handleSearch}>
+            Serach
+          </button>
+        </div>
+        {!!projects?.length && (
+          <div className='project-list'>
+            {projects?.map((project) => {
+              const { id, name, description, creator, skills, tags } = project;
+              return (
+                <div
+                  key={`project-${id}`}
+                  className='project-card'
+                  onClick={() => navigateToProject(id)}
+                >
+                  <div className='custom-card-header'>
+                    <h3>{name}</h3>
+                    <h5>{creator}</h5>
```

Comment:
As mentioned previously, it's best to follow the order of heading tags and use styling to change the size of the text.